### ISSUE TITLE
pobtoken deck_spawn command with the -m flag

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#153]
I've been testing `pobtoken deck_spawn my_first_pod_deck -m 1.5 --number_of_decimals 3 -f 490000 -e 500000` command.

The creation of the deck went smoothly, it was created with `1991d3d80c5df0d7c7b90afd4acf67d398cf8868f3a48b7568a96871bcc6f8e7` transaction (you should see it on your side, I guess).

The I've burned coins using `pobtoken burn_coins 1 -i 1991d3d80c5df0d7c7b90afd4acf67d398cf8868f3a48b7568a96871bcc6f8e7` command (transaction `a45563c27360f25637f0cc12ca4d4d9b44849656ac3f56ce37125f220aa77331`).
And claimed them with `pobtoken claim 1991d3d80c5df0d7c7b90afd4acf67d398cf8868f3a48b7568a96871bcc6f8e7 a45563c27360f25637f0cc12ca4d4d9b44849656ac3f56ce37125f220aa77331` command (transaction `db3662131d5dbe31825cf72a3007580bcdbb60f798a419e81ebad7003d61b934`).
However my `token balances -t 1991d3d80c5df0d7c7b90afd4acf67d398cf8868f3a48b7568a96871bcc6f8e7` output is 
```
{'n3MtPoPaAREU5GEhAErBPuju83bVog2opz': 1.0}
```
So it seems like the multiplier values wasn't applied, unless I've done something wrong.
_
Update 16-07-2024
I've spawned another deck with the `pobtoken deck_spawn my_first_pod_deck -m 2 --number_of_decimals 1 -f 490000 -e 500000` command, burned 1 tSLM with the `71a7283f5915bcc15481b8911137827d54cf09c956b30d7a22bec044e6673ca8` transaction and claimed it with the `ce4b6f081141e54ff4a74372d78db0a7ad51ffa42c038fdde5c7a42896614187` transaction and my `token balances -t efb8c9aa6bdd5d48a711ccfae351d6e7ef2340c439b2cde7aa711d9047ddaf18` output is 2 tokens as expected.
So probably the fractional multiplier is not allowed. If so we should consider mentioning it in the help.

**Comment by d5000** [comment]
I originally wanted fractional multipliers to work, but they were a pain to implement correctly, so I desisted from it.

To the token you created, a multiplier of 1 was assigned (basically "no multiplier" thus, as you saw.)

I have now added a check to the attoken/pobtoken spawn functions to prevent people from creating decks with fractional multipliers.

I made some more small improvements in commit  (e.g. adding a debug flag, InsufficientFunds error will be shown in red), but in general this can be closed if everything works for you (on my side everything runs fine).
